pkg/static/login.html

```diff
@@ -30,9 +30,12 @@ <h1 id="brand" class="hide-before"></h1>
       <svg fill="currentColor" viewBox="0 0 512 512" aria-hidden="true">
         <path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h28.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z" />
       </svg>
-      <h2 id="login-error-message" class="pf-v5-c-alert__title">
-        <span class="noscript" translate="yes">Please enable JavaScript to use the Web Console.</span>
-      </h2>
+      <p id="login-error-title" class="pf-v5-c-alert__title"></p>
+      <div class="pf-v5-c-alert__description">
+        <p id="login-error-message">
+          <span class="noscript" translate="yes">Please enable JavaScript to use the Web Console.</span>
+        </p>
+      </div>
```

Comment:
As for layout, I would expect a block with this structure

- alert
   - icon
   - title
   - description (optional)

Then a CSS grid like this:

```css
.pf-v5-c-alert {
  display: grid;
  grid-template: "icon content" / auto 1fr;
  gap: 0.5rem 1rem;
  align-items: center;
}

.pf-v5-c-alert__title,
.pf-v5-c-alert__description {
  grid-column: content;
}
```

It basically sets up a grid to have two columns, where the first is automatically sized and the other column is sized to the rest of the space.

Details: The icon goes into the first available grid cell, which is sized to auto and is called "icon" (which doesn't matter really, as we're not referring to it otherwise; it's implicitly placed — we could explicitly tell it to go there, but it's fine as-is as long as it's the first child of the grid anyway). Then the title and description are told to go into the "content" column, which is the remainder space. Since there are only two columns, the third child of the grid is on another row (and would go to the icon column, except we told it to go to the content column instead). The gap adds space between everything (both vertical and horizontal gaps as shorthand), and that vertical gap it makes the description optional, as it's unused if there's no second row with the description element.

You can see a full demo @ https://codepen.io/garrett/pen/YzmVmdK

(This uses the default PF5 HTML for the alert, not adjusted or anything, so it should theoretically be mostly a drop-in to our CSS.)

Reply:
It's not 100% perfect yet, but

> Garrett: it's mostly good enough, definitely better than what was there before
> we could also do that in a cosmetic follow-up

So let's get that in, and handle the rest in a follow-up if we care.